packages/blitz/src/utils/run-prisma.ts

```diff
@@ -1,27 +1,32 @@
-import which from "npm-which"
 import pEvent from "p-event"
 import {Readable} from "stream"
 import {spawn} from "cross-spawn"
+import resolveFrom from "resolve-from"
+import {log} from "../logging"
 
 export const runPrisma = async (args: string[], silent = false) => {
-  const prismaBin = which(process.cwd()).sync("prisma")
+  const prismaBin = resolveFrom.silent(process.cwd(), "prisma")
 
-  const cp = spawn(prismaBin, args, {
-    stdio: silent ? "pipe" : "inherit",
-    env: process.env,
-  })
-
-  const cp_stderr: string[] = []
-  if (silent) {
-    cp?.stderr?.on("data", (chunk: Readable) => {
-      cp_stderr.push(chunk.toString())
+  if (prismaBin) {
+    const cp = spawn(prismaBin, args, {
+      stdio: silent ? "pipe" : "inherit",
+      env: process.env,
     })
-  }
 
-  const code = await pEvent(cp, "exit", {rejectionEvents: []})
+    const cp_stderr: string[] = []
+    if (silent) {
+      cp?.stderr?.on("data", (chunk: Readable) => {
+        cp_stderr.push(chunk.toString())
+      })
+    }
+
+    const code = await pEvent(cp, "exit", {rejectionEvents: []})
 
-  return {
-    success: code === 0,
-    stderr: silent ? cp_stderr.join("") : undefined,
+    return {
+      success: code === 0,
+      stderr: silent ? cp_stderr.join("") : undefined,
+    }
+  } else {
+    log.error("Prisma bin not found")
```

Comment:
It should return in each case, as other usages expect this function to return a value, e.g.:
<img width="430" alt="CleanShot 2022-09-23 at 15 44 04@2x" src="https://user-images.githubusercontent.com/9019397/191974426-2ea94ce3-6af1-4977-ae81-c914da2bf8e2.png">

Not sure why CI didn't catch this.